prime-dotnet-webapi/ViewModels/Agreements/OrgAgreementRazorViewModel.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace Prime.ViewModels.Agreements
+{
+    public class OrgAgreementRazorViewModel
+    {
```

Comment:
Is there a need to include `Razor` in the classname?  Also seems inconsistent compared to other ViewModel classes

Reply:
This is a ViewModel used by Razor specifically. It is used to render the Org Agreement.